src/main/kotlin/no/nav/k9/domene/repository/ReservasjonRepository.kt

```diff
@@ -217,8 +217,8 @@ class ReservasjonRepository(
         tx.run(
             queryOf(
                 """
-                        insert into reservasjon as k (id, data)
-                        values (:id, :dataInitial :: jsonb)
+                        insert into reservasjon as k (id, data, opprettet)
+                        values (:id, :dataInitial :: jsonb, current_timestamp)
                         on conflict (id) do update
                         set data = jsonb_set(k.data, '{reservasjoner,999999}', :data :: jsonb, true)
```

Comment:
ser ut som uuid for reservasjoner er id'en til oppgaven. Så hvis en reservasjon blir oppdatert eks. bytter person, vil da opprettet tid bli overskrevet? (Litt mye postgres json magi som jeg ikke husker detaljer på 😅 ) .

Reply:
Godt poeng. Må nesten teste det for å være sikker selv 😄 Litt usikker på om vi bare setter data 'on conflict' eller om vi da også overskriver current_timestamp